osf/models/node.py

```diff
@@ -2061,12 +2059,13 @@ def fork_node(self, auth, title=None, parent=None):
 
         # Clone each log from the original node for this fork.
         self.clone_logs(forked)
-        forked.refresh_from_db()
 
         # After fork callback
         for addon in original.get_addons():
             addon.after_fork(original, forked, user)
 
+        forked.save()
```

Comment:
Did you take into consideration the comment on the line right below the moved save? Is this no longer the case?

`# Need to call this after save for the notifications to be created with the _primary_key`